### PR TITLE
Anthony green

### LINKED ISSUE #1015: Anthony Green bio submission
End user submitted issue from page: [/collect/person/](https://history.newtheatre.org.uk/collect/person/)
---
# 'Submit an almni bio' form submission

Field | Data
----- | ----
Name | Anthony Green
Grad Year | 1995
Course | Law (LLB)

## Bio1 (Time at theatre)

Will do. More to come. :)

## Bio2 (Post-graduation)

Will send soon. gotta eat. best :)

Field | Data
------| ----
Checked careers | Director, Producer, Actor, 
Other careers | 

## Shows

John Proctor - The Crucible - 1993  (not sure of month) directed by Peter Collins.
Marwood (I) - Withnail and I 1994 (not sure of month) directed by Sam and Jim (surnames to come)

Director - Plastic Madonna by Richard Davidson - St Columba's by the Castle for New Theatre Edinburgh 1994
Director - Conventional Demons by Richard Davidson - Pleasance for New Theatre Edinburgh Fringe Festival 1995

Akaky Akakievich The Overcoat Directed by Tim Wood - St Columba's by the Castle Edinburgh 1993
Sheriff Baldwin Rawel Storybook by Richard Davidson 1993 Directed by Richard Davidson for Nottingham New Theatre. Winner of Fringe First. St Columba's by the Castle.
Metamorphosis. by Franz Kafka. The Father. 1993 Edinburgh Fringe. St Columba's by the Castle.

thanks - more to come. best. Anthony

## Committees

President of New Theatre 1994-1995

## Contact Preferences

Are we allowed to facilitate contact to this alumnus? **Yes**

## Attempted File Generation

```
---
title: Anthony Green
course:
  - Law (LLB)
graduated: 1995
contact_allowed: true
careers:
  - Director
  - Producer
  - Actor


links: *fill me out

award: *fill me out

---

Will do. More to come. :)

Will send soon. gotta eat. best :)

```
**

### PR DESCRIPTION
**Pull request by nhyty**
will close #1015 . not yet finished though

### CODE REVIEW

**Comment by BenWoodford76** [comment]
Will review when @nhyty has finished additions

**Comment by nhyty** [comment]
@wjdp0

**Comment by wjdp** [comment]
Hmm, will have to have a look

File: _venues/pleasance_courtyard.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (wjdp) [inline _venues/pleasance_courtyard.md]: The filename wants to be `pleasance-courtyard.md` with a hyphen.

File: _venues/st_columbas_by_the_castle.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (wjdp) [inline _venues/st_columbas_by_the_castle.md]: Same here.

**Pull request merged**